references/video_classification/train.py

```diff
@@ -313,10 +332,10 @@ def main(args):
     print(f"Training time {total_time_str}")
 
 
-def parse_args():
+def get_args_parser(add_help=True):
```

Comment:
We change this so it become more similar to classification script and it is more compatible with `submitit` script

Reply:
Good call.